### DISCUSSION TITLE
Github Free And paid plan

### DISCUSSION THREAD

**Discussion by sarcasticoder** [community/community#142490] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I have a couple of questions on CICD pipeline minutes.

1. As per GitHub they will provide 3000 CICD minutes, But is it repo-wise or for the entire organization?
2. If I exhaust these minutes in free plan will I be able to buy few more minutes?

**Comment by thomassiard** [comment]
Hi! I'll try to clarify those questions about GitHub Actions minutes for the free plan:

Are the 3,000 CI/CD minutes repo-wise or for the entire organization?

The 3,000 minutes are allotted at the organization level (or account level if it's an individual account). So, if you have multiple repositories under one organization, they will share the same pool of 3,000 minutes per month.
Can I buy more minutes if I exhaust them on the free plan?

Yes, GitHub offers additional minutes as an add-on. Once the 3,000 free minutes are used up, you can purchase extra minutes. These are billed based on usage, and you can add them under Settings > Billing in your GitHub account.
Hope this helps! Let me know if there’s anything else you’re wondering about. 😊
